README.md

```diff
@@ -201,25 +214,30 @@ An example of such a mask would be `3XX, 4XX, 5XX`. In this case, all 300s, 400s
+| url                                        | required | The base URL that should be use for HTTP requests. For example _http://localhost:8080/client_.                                                                                                     |
+| format                                     | required | Specify what format to use.                                                                                                                                                                        |
+| insert-method                              | optional | Specify which HTTP method to use in the request. The value should be set either to `POST` or `PUT`.                                                                                                |
+| sink.batch.max-size                        | optional | Maximum number of elements that may be passed in a batch to be written downstream.                                                                                                                 |
+| sink.requests.max-inflight                 | optional | The maximum number of in flight requests that may exist, if any more in flight requests need to be initiated once the maximum has been reached, then it will be blocked until some have completed. |
+| sink.requests.max-buffered                 | optional | Maximum number of buffered records before applying backpressure.                                                                                                                                   |
+| sink.flush-buffer.size                     | optional | The maximum size of a batch of entries that may be sent to the HTTP endpoint measured in bytes.                                                                                                    |
+| sink.flush-buffer.timeout                  | optional | Threshold time in milliseconds for an element to be in a buffer before being flushed.                                                                                                              |
+| gid.connector.http.sink.request-callback   | optional | Specify which `HttpSinkPostRequestCallback` implementation to use. By default, it is set to `slf4j-logger` corresponding to `Slf4jHttpSinkPostRequestCallback`.                                    |
+| gid.connector.http.sink.error.code         | optional | List of HTTP status codes that should be treated as errors by HTTP Sink, separated with comma.                                                                                                     |
+| gid.connector.http.sink.error.code.exclude | optional | List of HTTP status codes that should be excluded from the `gid.connector.http.sink.error.code` list, separated with comma.                                                                        |
 
```

Comment:
sorry what was changed here?

Reply:
Similarly, added `gid.connector.http.sink.error.code`, `gid.connector.http.sink.error.code.exclude` (both weren't listed before), and `gid.connector.http.sink.request-callback` (as it is a new one).